GameDefinition/PLAYING.md

```diff
@@ -527,6 +539,99 @@ influence the speed of HP attrition and if it outweighs the benefits
+the ground (although the stash has a physical location on the map). You
+cannot operate the items from the stash, as if these items are just on
+the ground directly. To collect an item from the stash or add it there,
+you should enter the stash management menu by pressing the 'I'.
+
+Obviously, if the item is beneficial (a piece of armor, weapon, or a bonus
+item), it is reasonable to put it into a particular character's outfit.
+However, you should remember that the outfit is limited to any 10 (ten)
+items only. On the contrary, if the item is not required directly (ranged
+projectiles, medkits, ampoules with drugs, or food), it may be stored in
+the shared stash.
+The shared stash is of unlimited size. It has a physical location on
+the map and may be looted by your enemies. So, it is wise to keep it safe
+or change its location depending on the current situation. It would be
+a pity to lose your belongings, valuables, and powerful projectiles by not
+taking care of them. Enemy factions may have their stashes on the map,
+and you may loot their stashes too. Enemy units may have items in their
+outfits, and sometimes they may have some items stashed on the ground.
+To relocate the stash, first, you need to remove ALL of the items from it
+anywhere you want to make it empty. Then, any stashed item (or a pile
+of items) will automatically become the new stash. The process is a bit
+long, so you should plan stash relocation in advance.
+You should keep your eye on the item management stat in the character menu
+'#'. If this stat is too low (which may be caused by injury or panic), you
+will be unable to work with items either in the character's outfit or in
+the shared stash.
+
+To open the pointsman's outfit, press 'O' (if you want to see the pointsman's
+personal inventory outfit and manipulate items stored there) or 'i' (if
+you want to stash something in the shared inventory from the pointsman's
+outfit). NA outfit).
+To open the shared stash, press 'I'.
+To open the whole team's belongings (personal and stashed), press 'T'.
+To manage items on the ground (trigger or fling it directly not putting
+it into any inventory to save time), press 'G'.
+
+To pick up an item from the ground to the pointsman's outfit, press 'g'.
+To pick up an item from the ground to the stashed inventory, press ','.
+
+When you are managing items in the inventory or on the ground, you should
+choose the item with the LMB or arrows and 'RET' buttons to get the item
+management menu. You may want to choose all items from the particular location
+by pressing '*'.
+
+To drop an item on the ground from the inventory (either stashed or from
+the personal outfit), press 'r'.
+To trigger the item's properties (inject an ampoule, activate a chip, eat
+a piece of food), press 't'.
+To throw (fling) the item somewhere, press 'f'.
+To put the item into the particular outfit (personal or shared), press 'o'
+(personal outfit) or 'i' (shared inventory).
```

Comment:
Well, I thought it would be reasonable. But yes, it's quite a lot, and it duplicates the in-game help. For me, IMHO, it would be good.
But if it is not, I will remove. It just took for me a while to get used to the system. I'm more used to how it is arranged in Nethack or some other games like that. So, I wrote it in the way, as if I was writing to me from the past :)

But this fragment is redundant indeed. So,shall we remove it?

Reply:
I'm fine removing. However, I'd like to understand why the help screen is was not enough in your case. The commands you list here are beyond the minimal command set listed first in the help screen. Perhaps it's too small a set? Are you using 'o' and 'i' often? Do you think they are important for a newcomer? How did you come up with this particular list and not any wider subset?

Reply:
Well, there are many commands about dealing with the inventory, and playing with the personal/shared inventories is an important part of the game. It took a while to learn these things, and still, I am not used to some of the commands (and the 'TAB' command may crash the game sometimes).
I routinely use the 'i' and 'I' commands to navigate items between the outfits. I rarely use the 'TAB' command. Maybe, I'll start using it more often to look for that bug.

Reply:
I have deleted this fragment. It is available in the in-game help. Thus, indeed, no need to duplicate.